### ISSUE TITLE
[Feature Request] Non-interactive label chips

### ISSUE THREAD

**Issue by mitar** [vuetifyjs/vuetify#10115]
### Problem to solve
This is a followup to https://github.com/vuetifyjs/vuetify/issues/3817. Now, when I have a chip like:

```
<v-chip label>Foobar</v-chip>
```

If I hover with the mouse over it, it still changes the gray-level of the background. This makes it look like it is actionable. Still confusing.

### Proposed solution
If I do:

```
<v-chip label :input-value="true">Foobar</v-chip>
```

It makes it not change on hover the color. I think this should be the default for labels at least.

**Comment by jacekkarczmarczyk** [comment]
It sounds more like https://github.com/vuetifyjs/vuetify/issues/4656, imho `readonly` prop would a good choice here (i wouldn't make it default though)

**Comment by mitar** [comment]
#4656 was closed as a duplicate of #3817. And I think this one is a followup to #3817. :-)

**Comment by johnleider** [comment]
Thank you for the Feature Request and interest in improving Vuetify. After careful consideration the team has decided that this is not functionality that we are looking to implement at this time.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by ikovac** [comment]
@mitar This solution:
`<v-chip label :input-value="true">Foobar</v-chip>`
actually changes chip color by applying hover state color on a chip. Color won't be changed on hover but the initial color will be changed. That may result in unexpected behavior and shouldn't be used by the default.

**Comment by johnleider** [comment]
fixed by #13192

If you have any questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
